client/public/css/base.css

```diff
@@ -1,3 +1,29 @@
 .poemText {
     white-space: pre;
 }
+
+@media print {
+    body * {
+        visibility: hidden;
+        font-family: Georgia, serif;
+        background: none;
+        color: black;
+    }
+    .poem * {
+        visibility: visible;
+    }
+    .poem {
+        height: 100%;
+        width: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+        padding: 15px;
+        font-size: 14px;
+        line-height: 18px;
+    }
+    .poemTitle, p {
```

Comment:
I think it would be cleaner to give the author content its own CSS class instead of targeting `p`.